src/sshagent/KeeAgentSettings.h

```diff
@@ -26,6 +26,27 @@ class EntryAttachments;
 class OpenSSHKey;
 class QXmlStreamReader;
 
+class KeeAgentKeySpec
+{
+public:
+    bool operator==(const KeeAgentKeySpec& other) const;
+
+    QString keyblob;
+    bool is_ca;
+};
+
+class KeeAgentDestinationConstraint
+{
+public:
+    bool operator==(const KeeAgentDestinationConstraint& other) const;
+
+    QString fromHost;
+    QList<KeeAgentKeySpec> fromHostKeys;
+    QString toUser;
+    QString toHost;
+    QList<KeeAgentKeySpec> toHostKeys;
+};
```

Comment:
@droidmonkey before proceeding with polishing, may I ask a couple of dump questions? I'm not a C++ developer, sadly.

So I need this two helper classes as `KeeAgentSettings` holds a list of `KeeAgentDestinationConstraint` which itself holds two lists of `KeeAgentKeySpec`.

Q:
- Should each of them go into a separate header and source files?
- Should I write getters and setters for the members of the helper classes? (and if so,  I guess I should also add the `m_` prefix to the members as well)

Reply:
They should be structs defined within the KeeAgentSettings class

Reply:
Thank you 👍